typings/index.d.ts

```diff
@@ -64,19 +65,22 @@ interface GetByAPI {
 }
 
 interface QueryByAPI {
-  queryByText: (name: string | RegExp) => ReactTestInstance | null;
+  queryByText: (name: string | RegExp, options?: TextMatchOptions) => ReactTestInstance | null;
```

Comment:
RTL in their docs (but curiously not their ts typesing code) have a type TextMatch = string | RegExp, maybe we could use the same.